From 1a57dc41a8a4105781495b0d2956ee69cdd605ba Mon Sep 17 00:00:00 2001
From: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date: Fri, 27 Mar 2020 12:45:34 +0200
Subject: [PATCH 70/89] platform/x86: surface3_power: Drop unused structure
 definition

As reported by kbuild bot the struct mshw0011_lookup in never used.
Drop its definition for good.

Reported-by: kbuild test robot <lkp@intel.com>
Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
---
 drivers/platform/x86/surface3_power.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/platform/x86/surface3_power.c b/drivers/platform/x86/surface3_power.c
index 42331d178674..9cdae35c9802 100644
--- a/drivers/platform/x86/surface3_power.c
+++ b/drivers/platform/x86/surface3_power.c
@@ -56,13 +56,6 @@ struct mshw0011_data {
 	s32			full_capacity;
 };
 
-struct mshw0011_lookup {
-	struct mshw0011_data	*cdata;
-	unsigned int		n;
-	unsigned int		index;
-	int			addr;
-};
-
 struct mshw0011_handler_data {
 	struct acpi_connection_info	info;
 	struct i2c_client		*client;
-- 
2.30.2

